src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -69,16 +71,22 @@
     private GenericContainer<?> cntn;
 
     /**
-     * Artipie server port.
+     * URL 'http://host:port/my-nuget'.
      */
-    private int port;
+    private String url;
+
+    /**
+     * Package name in the temporary directory.
+     */
+    private String pckg;
```

Comment:
@genryxy value for this field is always `"newtonsoft.json"`, so this field should not be mutable. Please replace it with constant